vmware-vra.gemspec

```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "pry",     "~> 0.10"
   spec.add_development_dependency "rake",    "~> 10.0"
   spec.add_development_dependency "rspec",   "~> 3.0"
-  spec.add_development_dependency "webmock", "~> 1.21"
```

Comment:
I assume you ran into the Ruby 2.6 issues here?

Reply:
Yes, I bumped Webmock to ~3.5 so that it all now works in Travis.